core/templates/domain/story/story-node.model.ts

```diff
@@ -93,7 +93,7 @@ export class StoryNode {
     return this._explorationId;
   }
 
-  setExplorationId(explorationId: string): void {
+  setExplorationId(explorationId: string | null): void {
```

Comment:
Exploration id is also of type string and none in backend. Please take a look [here](https://github.com/oppia/oppia/blob/7aea401b372b38a66442e447332e4e06ca5880f6/core/domain/story_domain.py#L210)

Reply:
Ok!